engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -1991,9 +2034,16 @@ private void verifyLiveMigrationMapForKVM(Map<VolumeInfo, DataStore> volumeDataS
                 throw new CloudRuntimeException("Destination storage pool with ID " + dataStore.getId() + " was not located.");
             }
 
-            if (!destStoragePoolVO.isManaged()) {
-                throw new CloudRuntimeException("Migrating a volume online with KVM can currently only be done when moving to managed storage.");
-            }
+            throwExeptionIfCannotLiveMigrateOnANonManagedStorage(destStoragePoolVO);
+        }
+    }
+
+    /**
+     * It throws a {@link CloudRuntimeException} if the destiny storage pool is not a managed storage pool.
+     */
+    protected void throwExeptionIfCannotLiveMigrateOnANonManagedStorage(StoragePoolVO destStoragePoolVO) {
```

Comment:
I only find one usage of this method, is it planned to be invoked from elsewhere? Can we also rename it to something like 'checkDestinationStoragePool' ?

Reply:
The plan was to extract it to a method so I could change in the NonManaged to not throw an exception, but to be honest, I could just remove the exception as if it is not a managed storage it should not execute it anyway.

Reply:
Removed that method @nvazquez. It does not make sense to have that `throwExeptionIfCannotLiveMigrateOnANonManagedStorage`. Thanks for pointing that!